handlers/easy_info/__init__.py

```diff
@@ -0,0 +1,12 @@
+from eventhandler import EventHandler
+
+
+class EasyInfoHandler(EventHandler):
+    def on_issue_labeled(self, api, payload):
```

Comment:
Should we have an `on_issue_opened` (similar to `on_pr_opened`)? Because, issues "could" be labeled at the time of their creation.